src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,49 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Cars {
+
+    private static final int FIRST_INDEX = 0;
```

Comment:
```suggestion
    private static final int WINNER_INDEX = 0;
```
상수 처리👍
다만, 조금 더 의미있는 이름을 사용해보는건 어떨까요?

Reply:
더 의미가 생긴거같습니다!